lib/ruby_lsp/ruby_lsp_rails/addon.rb

```diff
@@ -141,7 +144,7 @@ def register_additional_file_watchers(global_state:, outgoing_queue:)
                 register_options: Interface::DidChangeWatchedFilesRegistrationOptions.new(
                   watchers: [
                     Interface::FileSystemWatcher.new(
-                      glob_pattern: "**/*structure.sql",
+                      glob_pattern: "**/*structure.sql,**/*routes.rb,routes/**/*.rb",
```

Comment:
I think this should almost almost any way that people set up the Rails routes, but if this is a risk of a false positive if someone is using the term `routes` for something else in their app.

Reply:
Possibly:
```
dummy(dev)> Rails.application.routes.draw_paths.map(&:to_s)
=> 
["/Users/andyw8/.gem/ruby/3.3.0/gems/actionview-8.0.0.beta1/config/routes",
 "/Users/andyw8/.gem/ruby/3.3.0/gems/activestorage-8.0.0.beta1/config/routes",
 "/Users/andyw8/.gem/ruby/3.3.0/gems/actioncable-8.0.0.beta1/config/routes",
 "/Users/andyw8/.gem/ruby/3.3.0/gems/actionmailbox-8.0.0.beta1/config/routes",
 "/Users/andyw8/.gem/ruby/3.3.0/gems/actiontext-8.0.0.beta1/config/routes",
 "/Users/andyw8/src/github.com/Shopify/ruby-lsp-rails/test/dummy/config/routes",
 "/Users/andyw8/src/github.com/Shopify/ruby-lsp-rails/test/dummy/config/routes"]
```